RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* Promoting `unbatch` from experimental to core API.
```

Comment:
that a dataset change? We should be more specific which `unbatch`.

Reply:
Done.